py3status/modules/uptime.py

```diff
@@ -46,7 +45,7 @@
 }
 ```
 
-@author Alexis "Horgix" Chotard <alexis.horgix.chotard@gmail.com>, tobes, lasers
```

Comment:
I'd leave the email address here.

Reply:
Identical names. We should keep name or email, but not both.

Reply:
It is actually valid, it's a display name and email